tests/unit_tests/dataframe_test.py

```diff
@@ -0,0 +1,55 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+# pylint: disable=unused-argument, import-outside-toplevel
+from superset.dataframe import df_to_records
+from superset.typing import DbapiDescription
+
+
+def test_df_to_records(app_context: None) -> None:
```

Comment:
```suggestion
def test_df_to_records(_: None) -> None:
```

Reply:
We need `app_context` here to be named as is, since it's a [fixture](https://docs.pytest.org/en/latest/explanation/fixtures.html#about-fixtures) used by the test.